libafl/src/mutators/mopt_mutator.rs

```diff
@@ -340,7 +340,7 @@ impl MOpt {
             || (res + 1 < operator_num
                 && select_prob > self.probability_now[self.swarm_now][res + 1])
         {
-            return Err(Error::MOpt("Error in select_algorithm".to_string()));
+            return Err(Error::mopt("Error in select_algorithm".to_string()));
```

Comment:
Unrelated, but does an extra mopt error really make sense?

Reply:
It's not helpful.
This error message is copy-pasted from the thesis-author's repo.

Reply:
Removed it